### DISCUSSION TITLE
New Contributor Suggested Onboarding Tasks (Outreachy and GSOC)

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/caMicroscope#681] [Announcements]
Hi everyone! We've put a (draft) list of tasks which we think can help people get started with caMicroscope. It's here: https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md

Please let us know if you have any feedback/suggestions, especially if the list or something on it is unclear!

**Comment by keamybams** [comment]
Hello. Thank you. Will do.

**Comment by Khadijaarike** [comment]
Hello @birm , I am Khadijat, my outreachy initial application got approved, and I'm excited to dive into contributing to Camicroscope! I've checked out the task list and spotted a few things I'm interested in. Could you walk me through what I need to do next to get started?

**Reply by Shulamite54** [reply]
Hello @Brim, good afternoon, name is Precious, my initial application was approved and am excited to be here, please can you guide me through on what steps to start from. Thank you.

**Comment by Amiynarh** [comment]
Thank you @birm , I'll let you know if I have any questions

**Comment by We13b-MD** [comment]
Hello @birm i am Idundun Michael hoping to get the best from this project  and learn new thingsd and contribute as well I am also happy to meet everyone here

**Comment by birm** [comment]
I'll also put this here, a more general guide to contributing - http://camicroscope.org/contributing.html
Please let us know if detail is missing; we want these guides to evolve to be more helpful.

**Reply by We13b-MD** [reply]
ok thank you

**Reply by DIANAAJAMBO** [reply]
Thank you

**Reply by nwanduka** [reply]
Thank you so much @birm

**Reply by FlourishRalph** [reply]
Hi @birm, thank you for these resources. However, this link isn't opening on my end.

**Reply by Lahitan** [reply]
Thank you @birm

**Comment by Lahitan** [comment]
Hello @birm 
My name is Tawakalitu, and my initial application was approved, I am interested in contributing to Camicroscope.
I have knowledge in HTML, CSS and JavaScript.
I would like to know if there are languages I need to get familiar with, I look forward to gaining a lot of knowledge and experience through this project.

Thank you

**Reply by kemsguy7** [reply]
Hy Lahitan, kindly go through this docs : http://camicroscope.org/contributing.html, you'll find information on the existing codebase

**Comment by Shulamite54** [comment]
thank you @birm

**Comment by WanjaNjunge** [comment]
Thanks for these @birm, will reach out in case of anything

**Comment by soniyadotp** [comment]
Hi Everyone, I am Soniya from India, and my initial application was approved, I am interested in contributing to Camicroscope.
I have knowledge of HTML, CSS, JavaScript, React.js,  Next.js, Nodejs, Express, and MongoDB.
I would like to know if there are languages I need to get familiar with, I look forward to gaining a lot of knowledge and experience through this project and collaboration with the people.

**Reply by gbengaoluwadahunsi** [reply]
Hey @soniyaprasad77 , 
You are welcome, as now the frontend of the caMicroscope is developed in HTML, CSS, and JavaScript and one of the contributors has however suggested the frontend part might be moved to modern frontend frameworks.

I said all to say, you are well equipped to make your stellar contributions.

**Comment by blessdamy** [comment]
Hello everyone. 
Hello @birm It's a privilege to be here. I sincerely  hope I am able to learn and make valuable contributions to this project.

**Comment by barchakuz** [comment]
Hello there, i am behroze aslam, outreachy applicant 2024, i am excited to contribute on this project. i am game developer and front end developer with good grip on C#,C,HTML,CSS,JS,REACT,PHP and LARAVEL. Please infrom me i am too late start working on this project. I would to be part of this project/ Thanks

**Comment by gbengaoluwadahunsi** [comment]
Hello, you are gladly welcome

You are never late here

Try to go through the contribution ReadMe file and get your hands wet

Also if you feel some of the tasks have been accomplished, you can propose
a new features or features.

Happy contributions
Hurray.

**Reply by blessdamy** [reply]
@birm please I still face issues with my docker not running after giving all permissions and following necessaries intructions

**Reply by blessdamy** [reply]
I need help, It prevents me from progressing

**Reply by We13b-MD** [reply]
Have you configured your BIOS to enable virtual machine

**Reply by blessdamy** [reply]
yes, when I checked, it was already checked @We13b-MD

**Reply by We13b-MD** [reply]
What system are you using HP or dell if Hp restart your system when it restarts keep pressing F10 unto the bios setup comes on you can configure your virtual machine from there configuring your virtual machine from settings might not work. I hope this helps

**Reply by barchakuz** [reply]
@blessdamy  could you explain more details about issue, i might help you

**Comment by blessdamy** [comment]
@We13b-MD I use dell and virtual machine has been enabled

**Reply by We13b-MD** [reply]
Virtual machine should be enabled also in your bios setup
